tools/clang/test/HLSL/workgraph/node_track_rw_input_sharing.hlsl

```diff
@@ -0,0 +1,32 @@
+// RUN: %clang_cc1 -HV 2021 -verify %s
```

Comment:
verify test should be OK to move to SemaHLSL/hlsl/workgraph as lit test since it not run hash stability test.